### ISSUE TITLE
android: accessibility isn't working

### ISSUE THREAD

**Issue by vortex1024** [tailscale/tailscale#1004]
**Describe the bug**

when trying to navigate the app with the talkback screen reader, it doesn't see any controls. 
**To Reproduce**
1. go to settings\accessibility and enable the talkback screen reader
2. open tailscale, first touching the items to get focus on tehm, and then double tapping to actually open them
3. try to touch the ui controls or swipe tthrough them
4. talkback doesn't read anything

**Expected behavior**
talkback should read the UI contorls, text and role (button, list etc)

**Screenshots**

**Version information:**
 - Device: sony xz1 compact
 - OS: android
 - OS version: 9
 - Tailscale version: latest

**Additional context**
Talkback is a screen reader, an app which reads aloud UI widgets so blind people can use them. for info about how to make an app accessible, see here:
https://developer.android.com/guide/topics/ui/accessibility/apps
Since I am also a developer, I tried to quicly look in the source to try to identify the problem. I wasn't able to, I am nnot sure where the android UI is initiated. I am not even sure it uses native controls. If applicable, each clickable control should have android:screenReaderFocusable set to true.
Please let me know if you need any more info.

**Comment by eliasnaur** [comment]
Thank you for the report. I'll take a look.

**Comment by bradfitz** [comment]
Any update, @eliasnaur?

**Comment by eliasnaur** [comment]
Not yet, sorry. I'm still working on compatibility with lowest-spec phones (https://github.com/tailscale/tailscale/issues/1008). It's next in my queue, after the simpler #1401.
